includes/Editor.php

```diff
@@ -0,0 +1,264 @@
+<?php
+/**
+ * Class editor.
+ *
+ * Editor class, with output buffer.
+ *
+ * @package   Google\Web_Stories\Traits
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+namespace Google\Web_Stories;
+
+use Google\Web_Stories_Dependencies\AmpProject\Dom\Document;
+
+/**
+ * Class Editor
+ *
+ * @package Google\Web_Stories
+ */
+class Editor {
```

Comment:
I know it's still WIP, but Editor is very ambiguous.

It's a more suitable name for the stuff in `Story_Post_Type`.

Since this class tries to ensure cross-origin isolation, so maybe `Cross_Origin_Isolation` is more descriptive?

Reply:
Yeah, your right, I noted as much here - https://github.com/google/web-stories-wp/issues/5062#issuecomment-805744061

I think I had that in mind when I named it. I will change it.

Reply:
Changed.